### ISSUE TITLE
PackageControl List Two Packages Installed: "A File Icon" and "zzz A File Icon zzz"

### ISSUE THREAD

**Issue by Azakur4** [ihodev/a-file-icon#82]
I don't know is this is really a bug but when i list my package installed this to package appear: "A File Icon" and "zzz A File Icon zzz".

I don't know is this is right or this came from renamed project.

**Comment by ihodev** [comment]
Unfortunately this is a feature 😄 `zzz A File Icon zzz` folder automatically generates to apply the patches, however `tmPreferences` files have to be loaded first that's why the package's name starts with "A".

**Comment by diego898** [comment]
It might be worth adding this to the readme. Its strange that when you list packages in package control, `A File Icon` appears with all proper information under the title, while `zzz A File Icon zzz` says "No description provided" and "unknown version"

**Comment by ihodev** [comment]
It's already mentioned in [How It Works](https://github.com/ihodev/a-file-icon#how-it-works) section. However I think I'll provide some description for `zzz A File Icon zzz` in the next release.

**Comment by ihodev** [comment]
Ok, I've found the way how to fix it. It will be resolved in the next release.

**Comment by tr1s** [comment]
So 'zzz A File Icon zzz' should not be removed, correct?

**Comment by ihodev** [comment]
@tr1s Yes, it will be moved to the `Packages/User/{A File Icon}` in 4.0.0.
